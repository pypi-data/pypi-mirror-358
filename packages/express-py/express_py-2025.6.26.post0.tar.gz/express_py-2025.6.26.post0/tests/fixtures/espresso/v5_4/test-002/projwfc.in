&PROJWFC
    prefix = '__prefix__'
    outdir = '/home/exadmin/data/exadmin-default/new-job-aug-21st-2017-18-47-pm-cmF9STjvfyTkoJcte/outdir'
    degauss = 0.01
    deltaE = 0.05
/
