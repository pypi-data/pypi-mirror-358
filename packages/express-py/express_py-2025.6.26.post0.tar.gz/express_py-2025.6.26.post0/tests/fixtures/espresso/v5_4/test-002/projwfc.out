
     Program PROJWFC v.5.4.0 starts on 22Aug2017 at  1:48:20

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org",
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Serial version

     Reading data from directory:
     /home/exadmin/data/exadmin-default/new-job-aug-21st-2017-18-47-pm-cmF9STjvfyTkoJcte/outdir/__prefix__.save

   Info: using nr1, nr2, nr3 values from input

   Info: using nr1, nr2, nr3 values from input

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      = SLA  PW   PBE  PBE ( 1  4  3  4 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want


     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         745     583    169                13323     9409    1471


     Gaussian broadening (read from input): ngauss,degauss=   0    0.010000


     Calling projwave ....

     Atomic states used for projection
     (read from pseudopotential files):

     state #   1: atom   1 (Si ), wfc  1 (l=0 m= 1)
     state #   2: atom   1 (Si ), wfc  2 (l=1 m= 1)
     state #   3: atom   1 (Si ), wfc  2 (l=1 m= 2)
     state #   4: atom   1 (Si ), wfc  2 (l=1 m= 3)
     state #   5: atom   2 (Si ), wfc  1 (l=0 m= 1)
     state #   6: atom   2 (Si ), wfc  2 (l=1 m= 1)
     state #   7: atom   2 (Si ), wfc  2 (l=1 m= 2)
     state #   8: atom   2 (Si ), wfc  2 (l=1 m= 3)

 k =   0.0000000000  0.0000000000  0.0000000000
==== e(   1) =    -5.59666 eV ====
     psi = 0.498*[#   1]+0.498*[#   5]+
    |psi|^2 = 0.997
==== e(   2) =     6.27190 eV ====
     psi = 0.440*[#   3]+0.440*[#   7]+0.040*[#   4]+0.040*[#   8]+
    |psi|^2 = 0.962
==== e(   3) =     6.27192 eV ====
     psi = 0.481*[#   2]+0.481*[#   6]+
    |psi|^2 = 0.962
==== e(   4) =     6.27192 eV ====
     psi = 0.440*[#   4]+0.440*[#   8]+0.040*[#   3]+0.040*[#   7]+
    |psi|^2 = 0.962
==== e(   5) =     8.71291 eV ====
     psi = 0.483*[#   2]+0.483*[#   6]+
    |psi|^2 = 0.965
==== e(   6) =     8.71291 eV ====
     psi = 0.402*[#   4]+0.402*[#   8]+0.080*[#   3]+0.080*[#   7]+
    |psi|^2 = 0.965
==== e(   7) =     8.71292 eV ====
     psi = 0.402*[#   3]+0.402*[#   7]+0.080*[#   4]+0.080*[#   8]+
    |psi|^2 = 0.965
==== e(   8) =     9.41866 eV ====
     psi = 0.494*[#   1]+0.494*[#   5]+
    |psi|^2 = 0.988

 k =   0.2886751397  0.2041241219 -0.4999999735
==== e(   1) =    -3.29988 eV ====
     psi = 0.386*[#   1]+0.386*[#   5]+0.075*[#   2]+0.075*[#   6]+0.025*[#   3]+
          +0.025*[#   7]+0.013*[#   4]+0.013*[#   8]+
    |psi|^2 = 0.997
==== e(   2) =    -0.66264 eV ====
     psi = 0.250*[#   2]+0.250*[#   6]+0.119*[#   1]+0.119*[#   5]+0.083*[#   3]+
          +0.083*[#   7]+0.042*[#   4]+0.042*[#   8]+
    |psi|^2 = 0.988
==== e(   3) =     5.06314 eV ====
     psi = 0.360*[#   3]+0.360*[#   7]+0.082*[#   4]+0.082*[#   8]+0.052*[#   2]+
          +0.052*[#   6]+
    |psi|^2 = 0.989
==== e(   4) =     5.06315 eV ====
     psi = 0.357*[#   4]+0.357*[#   8]+0.112*[#   2]+0.112*[#   6]+0.025*[#   3]+
          +0.025*[#   7]+
    |psi|^2 = 0.989
==== e(   5) =     7.69786 eV ====
     psi = 0.342*[#   1]+0.342*[#   5]+0.080*[#   2]+0.080*[#   6]+0.027*[#   3]+
          +0.027*[#   7]+0.013*[#   4]+0.013*[#   8]+
    |psi|^2 = 0.924
==== e(   6) =     9.49433 eV ====
     psi = 0.256*[#   4]+0.256*[#   8]+0.095*[#   2]+0.095*[#   6]+0.031*[#   3]+
          +0.031*[#   7]+
    |psi|^2 = 0.766
==== e(   7) =     9.49434 eV ====
     psi = 0.267*[#   3]+0.267*[#   7]+0.084*[#   4]+0.084*[#   8]+0.032*[#   2]+
          +0.032*[#   6]+
    |psi|^2 = 0.766
==== e(   8) =    13.90052 eV ====
     psi = 0.046*[#   2]+0.046*[#   6]+0.019*[#   1]+0.019*[#   5]+0.015*[#   3]+
          +0.015*[#   7]+0.008*[#   4]+0.008*[#   8]+
    |psi|^2 = 0.175

 k =   0.0000000000 -0.6123724573  0.0000000000
==== e(   1) =    -3.29988 eV ====
     psi = 0.386*[#   1]+0.386*[#   5]+0.113*[#   4]+0.113*[#   8]+
    |psi|^2 = 0.997
==== e(   2) =    -0.66264 eV ====
     psi = 0.375*[#   4]+0.375*[#   8]+0.119*[#   1]+0.119*[#   5]+
    |psi|^2 = 0.988
==== e(   3) =     5.06314 eV ====
     psi = 0.494*[#   3]+0.494*[#   7]+
    |psi|^2 = 0.989
==== e(   4) =     5.06315 eV ====
     psi = 0.494*[#   2]+0.494*[#   6]+
    |psi|^2 = 0.989
==== e(   5) =     7.69786 eV ====
     psi = 0.342*[#   1]+0.342*[#   5]+0.120*[#   4]+0.120*[#   8]+
    |psi|^2 = 0.924
==== e(   6) =     9.49433 eV ====
     psi = 0.383*[#   2]+0.383*[#   6]+
    |psi|^2 = 0.766
==== e(   7) =     9.49434 eV ====
     psi = 0.383*[#   3]+0.383*[#   7]+
    |psi|^2 = 0.766
==== e(   8) =    13.90052 eV ====
     psi = 0.069*[#   4]+0.069*[#   8]+0.019*[#   1]+0.019*[#   5]+
    |psi|^2 = 0.175

 k =   0.2886751397 -0.4082483353 -0.4999999735
==== e(   1) =    -1.50837 eV ====
     psi = 0.281*[#   1]+0.281*[#   5]+0.108*[#   2]+0.108*[#   6]+0.072*[#   4]+
          +0.072*[#   8]+0.036*[#   3]+0.036*[#   7]+
    |psi|^2 = 0.994
==== e(   2) =    -1.50836 eV ====
     psi = 0.281*[#   1]+0.281*[#   5]+0.108*[#   2]+0.108*[#   6]+0.072*[#   4]+
          +0.072*[#   8]+0.036*[#   3]+0.036*[#   7]+
    |psi|^2 = 0.994
==== e(   3) =     3.41277 eV ====
     psi = 0.373*[#   3]+0.373*[#   7]+0.124*[#   2]+0.124*[#   6]+
    |psi|^2 = 0.996
==== e(   4) =     3.41277 eV ====
     psi = 0.332*[#   4]+0.332*[#   8]+0.124*[#   2]+0.124*[#   6]+0.041*[#   3]+
          +0.041*[#   7]+
    |psi|^2 = 0.996
==== e(   5) =     6.92240 eV ====
     psi = 0.171*[#   1]+0.171*[#   5]+0.123*[#   2]+0.123*[#   6]+0.082*[#   4]+
          +0.082*[#   8]+0.041*[#   3]+0.041*[#   7]+
    |psi|^2 = 0.835
==== e(   6) =     6.92240 eV ====
     psi = 0.171*[#   1]+0.171*[#   5]+0.123*[#   2]+0.123*[#   6]+0.082*[#   4]+
          +0.082*[#   8]+0.041*[#   3]+0.041*[#   7]+
    |psi|^2 = 0.835
==== e(   7) =    16.15049 eV ====
     psi = 0.303*[#   4]+0.303*[#   8]+0.116*[#   2]+0.116*[#   6]+0.044*[#   3]+
          +0.044*[#   7]+
    |psi|^2 = 0.925
==== e(   8) =    16.15049 eV ====
     psi = 0.341*[#   3]+0.341*[#   7]+0.116*[#   2]+0.116*[#   6]+0.006*[#   4]+
          +0.006*[#   8]+
    |psi|^2 = 0.925

 k =  -0.5773502794  0.2041242134  0.0000000000
==== e(   1) =    -3.29988 eV ====
     psi = 0.386*[#   1]+0.386*[#   5]+0.100*[#   3]+0.100*[#   7]+0.013*[#   4]+
          +0.013*[#   8]+
    |psi|^2 = 0.997
==== e(   2) =    -0.66265 eV ====
     psi = 0.333*[#   3]+0.333*[#   7]+0.119*[#   1]+0.119*[#   5]+0.042*[#   4]+
          +0.042*[#   8]+
    |psi|^2 = 0.988
==== e(   3) =     5.06315 eV ====
     psi = 0.494*[#   2]+0.494*[#   6]+
    |psi|^2 = 0.989
==== e(   4) =     5.06315 eV ====
     psi = 0.440*[#   4]+0.440*[#   8]+0.055*[#   3]+0.055*[#   7]+
    |psi|^2 = 0.989
==== e(   5) =     7.69787 eV ====
     psi = 0.342*[#   1]+0.342*[#   5]+0.106*[#   3]+0.106*[#   7]+0.013*[#   4]+
          +0.013*[#   8]+
    |psi|^2 = 0.924
==== e(   6) =     9.49434 eV ====
     psi = 0.340*[#   4]+0.340*[#   8]+0.043*[#   3]+0.043*[#   7]+
    |psi|^2 = 0.766
==== e(   7) =     9.49434 eV ====
     psi = 0.383*[#   2]+0.383*[#   6]+
    |psi|^2 = 0.766
==== e(   8) =    13.90053 eV ====
     psi = 0.061*[#   3]+0.061*[#   7]+0.019*[#   1]+0.019*[#   5]+0.008*[#   4]+
          +0.008*[#   8]+
    |psi|^2 = 0.175

 k =  -0.5773502794 -0.4082482439  0.0000000000
==== e(   1) =    -1.50837 eV ====
     psi = 0.281*[#   1]+0.281*[#   5]+0.144*[#   3]+0.144*[#   7]+0.072*[#   4]+
          +0.072*[#   8]+
    |psi|^2 = 0.994
==== e(   2) =    -1.50837 eV ====
     psi = 0.281*[#   1]+0.281*[#   5]+0.144*[#   3]+0.144*[#   7]+0.072*[#   4]+
          +0.072*[#   8]+
    |psi|^2 = 0.994
==== e(   3) =     3.41277 eV ====
     psi = 0.332*[#   4]+0.332*[#   8]+0.166*[#   3]+0.166*[#   7]+
    |psi|^2 = 0.996
==== e(   4) =     3.41277 eV ====
     psi = 0.498*[#   2]+0.498*[#   6]+
    |psi|^2 = 0.996
==== e(   5) =     6.92240 eV ====
     psi = 0.171*[#   1]+0.171*[#   5]+0.164*[#   3]+0.164*[#   7]+0.082*[#   4]+
          +0.082*[#   8]+
    |psi|^2 = 0.835
==== e(   6) =     6.92240 eV ====
     psi = 0.171*[#   1]+0.171*[#   5]+0.164*[#   3]+0.164*[#   7]+0.082*[#   4]+
          +0.082*[#   8]+
    |psi|^2 = 0.835
==== e(   7) =    16.15049 eV ====
     psi = 0.460*[#   2]+0.460*[#   6]+0.002*[#   4]+0.002*[#   8]+
    |psi|^2 = 0.925
==== e(   8) =    16.15049 eV ====
     psi = 0.307*[#   4]+0.307*[#   8]+0.153*[#   3]+0.153*[#   7]+0.002*[#   2]+
          +0.002*[#   6]+
    |psi|^2 = 0.925

Lowdin Charges:

     Atom #   1: total charge =   3.9588, s =  1.0503,
     Atom #   1: total charge =   3.9588, p =  2.9085, pz=  0.9695, px=  0.9695, py=  0.9695,
     Atom #   2: total charge =   3.9588, s =  1.0503,
     Atom #   2: total charge =   3.9588, p =  2.9085, pz=  0.9695, px=  0.9695, py=  0.9695,
     Spilling Parameter:   0.0103

     PROJWFC      :     1.00s CPU         1.07s WALL


   This run was terminated on:   1:48:21  22Aug2017

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
